sample-templates/deploy-sagemaker-mistral-model.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Deploy Mistral Model",
+  "description": "Deploy a model using a connector to SageMaker Mistral model",
+  "use_case": "PROVISION",
+  "version": {
+    "template": "1.0.0",
+    "compatibility": [
+      "2.12.0",
+      "3.0.0"
+    ]
+  },
+  "workflows": {
+    "provision": {
+      "nodes": [
+        {
+          "id": "create_mistral_connector",
+          "type": "create_connector",
+          "user_inputs": {
+            "name": "sagemaker: mistral",
+            "description": "Test connector for Sagemaker mistral model",
+            "version": "1",
+            "protocol": "aws_sigv4",
+            "credential": {
+              "access_key": "PUT_YOUR_ACCESS_KEY_HERE",
+              "secret_key": "PUT_YOUR_SECRET_KEY_HERE"
+            },
+            "parameters": {
+              "region": "us-east-1",
+              "service_name": "sagemaker"
+            },
+            "actions": [
+              {
+                "action_type": "predict",
+                "method": "POST",
+                "headers": {
+                  "content-type": "application/json"
+                },
+                "url": "https://runtime.sagemaker.us-east-1.amazonaws.com/endpoints/PUT_YOUR_ENDPOINT_HERE/invocations",
```

Comment:
this is a custom sagemaker endpoint, it shouldn't be limit to mistral?

Reply:
Can you recommend what to put here?   In the YAML I had comments about the user needing to change it, JSON doesn't allow comments.

Reply:
i think this can be something general like "Sagemaker Custom Model"